djaopsp/queries.py

```diff
@@ -889,16 +889,19 @@ def segments_as_sql(segments):
                 }
     return segments_query
 
-def get_portfolios_frozen_assessments(grantee=None, 
+def get_portfolios_frozen_assessments(grantees=None, 
```

Comment:
This implementation looks like a specific version of the implementation in `PortfolioAccessibleSamplesMixin.get_latest_frozen_by_accounts_by_period`.